Yosemite/Yosemite/Tools/Media/MediaType.swift

```diff
@@ -28,6 +28,19 @@ public enum MediaType {
     }
 
     private init(fileUTI: CFString) {
+        // Prior to iOS 14.4, video/webm and audio/webm were not natively supported and would
+        // have resulted as `self = .other` in the code below. iOS 14.4 added support for them,
+        // interpreting both as video types, but we don't want the app to allow users to upload
+        // them because WooCommerce/WordPress don't support that format, yet.
+        //
+        // The UT type identifier for both video/webm and audio/webm is "org.webmproject.webm"
+        //
+        // See https://github.com/woocommerce/woocommerce-ios/pull/4459/files#r656194065
+        guard "\(fileUTI)" != "org.webmproject.webm" else {
+            self = .other
+            return
+        }
```

Comment:
As discussed [here](https://github.com/woocommerce/woocommerce-ios/pull/4459#discussion_r656194065), even though iOS now supports WebM audio and video, we don't want the app to import them.

I initially thought to use a `if #available(iOS 14.4, *)` check, but then decided to use this one because:

1. It should work fine on Xcode < 12.5
2. We don't have any reason right now to support Xcode < 12.5

Reply:
@jkmassel @pmusolino @jaclync I refactored (= change of implementation, same behavior, as shown by the tests not having changed in this diff and still passing) as suggested by Paolo [here](https://github.com/woocommerce/woocommerce-ios/pull/4459#discussion_r656194065).

Can I ask you to have another look? Thanks!

Reply:
Looks good to me @mokagio, and thanks for explaining all the details in the comment 👍